docs/ORTModule_ModuleWithLoss_Wrapper.md

```diff
@@ -0,0 +1,94 @@
+# ONNX Runtime ModuleWithLoss Wrapper
+
+This document provides instructions on implementing a wrapper similar to the ModuleWithLoss Wrapper in Optimum. By implementing this wrapper, you can compute the loss inside ONNX Runtime (ORT), enabling you to leverage additional optimizations such as label sparsity optimization.
```

Comment:
the daption is not needed for all cases, shall we clarify in which case we need it? e.g. when 1). loss is not computed in the model forward path, 2). or model forward compute the loss but also return other outputs that are not needed for subsquent computation.

The impact of 1) is ORT cannot track loss compute in ONNX graph; impact of 2) is  ORT graph return more tensors that is needed, which will take unnecessary CUDA memory during backwards.

Reply:
Yeah I think giving context to why we're doing this and its impact will be useful.
Thanks, Will add this

Reply:
Thanks for making this change!